pages/services/cassandra/2.6.0-3.11.4/getting-started/index.md

```diff
@@ -0,0 +1,187 @@
+
+```
+$ dcos package install {{ model.serviceName }} --options=<options>.json
+```
+
+It is recommended that this custom configuration is stored in source control.
+
+For more information about building the `options.json` file, see the [DC/OS documentation](/1.11/deploying-services/config-universe-service/) for service configuration access.
+
+## Installation from the DC/OS Web Interface
+
+You can [install DC/OS Apache {{ model.techShortName }} from the DC/OS web interface](/1.11/deploying-services/install/). If you install DC/OS Apache {{ model.techShortName }} from the web interface, you must install the DC/OS Apache {{ model.techShortName }} CLI subcommands separately. From the DC/OS CLI, enter:
+```bash
+dcos package install {{ model.serviceName }} --cli
+```
+Choose `ADVANCED INSTALLATION` to perform a custom installation.
+
+## Integration with DC/OS access controls
+
+In Enterprise DC/OS 1.10 and above, you can integrate your SDK-based service with DC/OS ACLs to grant users and groups access to only certain services. You do this by installing your service into a folder, and then restricting access to some number of folders. Folders also allow you to namespace services. For instance, `staging/{{ model.serviceName }}` and `production/{{ model.serviceName }}`.
+
+Steps:
+
+1. In the DC/OS GUI, create a group, then add a user to the group. Or, just create a user. Click **Organization** > **Groups** > **+** or **Organization** > **Users** > **+**. If you create a group, you must also create a user and add them to the group.
+1. Give the user permissions for the folder where you will install your service. In this example, we are creating a user called `developer`, who will have access to the `/testing` folder.
+   Select the group or user you created. Select **ADD PERMISSION** and then toggle to **INSERT PERMISSION STRING**. Add each of the following permissions to your user or group, and then click **ADD PERMISSIONS**.
+
+   ```
+   dcos:adminrouter:service:marathon full
+   dcos:service:marathon:marathon:services:/testing full
+   dcos:adminrouter:ops:mesos full
+   dcos:adminrouter:ops:slave full
+   ```
+1. Install your service into a folder called `test`. Go to **Catalog**, then search for **{{ model.serviceName }}**.
+1. Click **CONFIGURE** and change the service name to `/testing/{{ model.serviceName }}`, then deploy.
+
+   The slashes in your service name are interpreted as folders. You are deploying {{ model.techShortName }} in the `/testing` folder. Any user with access to the `/testing` folder will have access to the service.
+
+<p class="message--note"><strong>NOTE: </strong>Services cannot be renamed. Because the location of the service is specified in the name, you cannot move services between folders. DC/OS 1.9 and earlier does not accept slashes in service names. You may be able to create the service, but you will encounter unexpected problems.</p>
```

Comment:
```suggestion
<p class="message--note"><strong>NOTE: </strong>Services cannot be renamed because the location of the service is specified in the name and you cannot move services between folders. DC/OS 1.9 and earlier does not accept slashes in service names. You may be able to create the service, but you will encounter unexpected problems.</p>
```